src/vm/eventtrace.cpp

```diff
@@ -1205,6 +1207,9 @@ void BulkComLogger::FlushRcw()
 
     ULONG result = EventWrite(Microsoft_Windows_DotNETRuntimeHandle, &GCBulkRCW, _countof(eventData), eventData);
     _ASSERTE(result == ERROR_SUCCESS);
+#else
+// UNIXTODO: "Eventing Not Implemented"
```

Comment:
This is a complicated scenario where currently the VM is directly emitting to the event stream, I have not found a straight forward way to abstract it. We will enable them once we finalize how such events are going to be read by the consumer